### PR TITLE
Automated missing Settings tests for Technology

### PR DESCRIPTION
**Pull request by naujalyte**

### CODE REVIEW

File: test/qa/test_settings.py [hunk 1]
```diff
@@ -94,3 +94,34 @@ def test_set_obfuscate_server_obfuscation_mismatch(obfuscateInitialState, server
     assert errorMessage in str(ex.value)
 
     sh.nordvpn.set.autoconnect.off()
+
+
+@pytest.mark.parametrize("tech,proto,obfuscated", lib.TECHNOLOGIES_BASIC2 + lib.TECHNOLOGIES_BASIC1)
+@pytest.mark.flaky(reruns=2, reruns_delay=90)
```
Comment (mariusSincovici) [inline test/qa/test_settings.py:100]: Why is this needed, isn't it always working?

**Review by mariusSincovici — COMMENTED** [review]
Not sure if it was intended, but it I think it can happen that the first run in each test is not doing anything, depending on the app existing settings and the test value.
e.g. when changing the technology, if `NORDLYNX` is already set, setting it again will have no impact.

File: test/qa/test_settings.py (hunk 1 above)
Comment (EmilijusS) [reply test/qa/test_settings.py:100]: We use these `flaky` decorators on tests that do connects because sometimes they just fail (well not "just", but it's complicated). But here you are right, I also think this is unnecessary. Timeout decorator below as well. FYI explicit timeout decorator is used with flaky decorator because the default timeout method in our tests just kills the python process while with a softer timeout the tests would be left hanging sometimes. But in this case because we need to restart a test we want a softer timeout, so an explicit timeout decorator is used with a lower timeout value than the global timeout.

**Review by EmilijusS — APPROVED** [review]

**Review by mariusSincovici — APPROVED** [review]

File: test/qa/test_settings.py (hunk 1 above)
Comment (naujalyte) [reply test/qa/test_settings.py:100]: "e.g. when changing the technology, if NORDLYNX is already set, setting it again will have no impact." I don't think that this should be happening since all of these tests start from changing the technology from the default Nordlynx to OpenVPN

**Comment by naujalyte** [comment]
> "e.g. when changing the technology, if NORDLYNX is already set, setting it again will have no impact." 

I don't think that this should be happening since all of these tests start from changing the technology from the default Nordlynx to OpenVPN

File: test/qa/test_settings.py (hunk 1 above)
Comment (naujalyte) [reply test/qa/test_settings.py:100]: Thanks! Will remove those from tests that do not do connects

**Review by mariusSincovici — APPROVED** [review]
